src/generic_core/remote-instance.ts

```diff
@@ -184,8 +185,10 @@ export var remoteProxyInstance :RemoteInstance = null;
         }
 
         // Create a new RtcToNet instance each time a new round of client peer
-        // messages begins.
-        if (signalFromRemote.first) {
+        // messages begins. The type field check is so pre-bridge,
```

Comment:
Done.

Yep. Ideally, UI should not be inspecting signalling messages; seems like that ship has sailed.